framework/decode/vulkan_ascii_consumer_base.cpp

```diff
@@ -39,16 +39,15 @@ void VulkanAsciiConsumerBase::Initialize(FILE* file)
 {
     assert(file);
     file_ = file;
-    fprintf(file_, "{");
 }
 
 void VulkanAsciiConsumerBase::Destroy()
 {
     if (file_ != nullptr)
     {
-        fprintf(file_, "\n}\n");
         file_ = nullptr;
     }
```

Comment:
`file_ = nullptr;` would suffice.

Reply:
@lunarpapillo  Replace the whole set of 5 lines in the original file with `file_ = nullptr;` now that we don't need to use `file_`?

Reply:
Just saw this and pushed an update accordingly.